service/policy/objects.proto

```diff
@@ -220,7 +220,7 @@ message SubjectConditionSet {
    and the external_value is the value selected by the external_selector_value on that Entity Representation (Subject Context). These mirror the Condition.
 */
 message SubjectProperty {
-  string external_selector_value = 1 [(buf.validate.field).required = true];
```

Comment:
I'm working on the `otdfctl` implementation of this API and realized the protovalidate below on line 224 that a field `external_value` must be specified breaks our desired `MatchSubjectMappingsRequest` implementation where only selectors are included for consideration (not values). The CLI must add a dummy value like `ExternalValue: "abc"` for a proto message to pass validation.

It looks like the proto message `SubjectProperty` is only used for this API, so perhaps we should remove this validation in this PR so the API can be called without extra work on clients for it to compile.